src/plugins/adsEverywhere/index.ts

```diff
@@ -0,0 +1,103 @@
+        { title: "10 Insane Facts About the Deep Ocean", image: "https://miro.medium.com/v2/resize:fit:1400/1*zqLjq7hwrduAoTJGeDqgag.jpeg" },
+        { title: "This Hack Will Change the Way You Use Your Phone", image: "https://ifixscreens.com/storage/2023/08/How-To-Protect-Your-Phone-From-Hackers-Smartphone-Hacking-Prevention.webp" },
+        { title: "The Best Movies You’ve Never Heard Of", image: "https://ecwpress.com/cdn/shop/products/9781550225907_1024x1024.jpeg?v=1479931324" },
+        { title: "How This Small Town Turned Into a Tourist Hotspot", image: "https://blog.sheswanderful.com/wp-content/uploads/2020/09/pexels-photo-4473398.jpeg" },
+        { title: "The Secret Lives of Billionaires", image: "https://i.ytimg.com/vi/f_loexi83Mg/maxresdefault.jpg" },
+        { title: "This Ancient Technique is Helping People Sleep Better", image: "https://physioentrust.com/wp-content/uploads/2021/09/maxresdefault-1024x688.jpg" },
+        { title: "DougDoug MugMug", image: "https://gist.github.com/user-attachments/assets/f51072fc-18df-48cb-8b00-ab76232a1b59", href: "https://dougdoug.shop" }
+    ];
+
+    function randomize(array) {
+        for (let i = array.length - 1; i > 0; i--) {
+            const swapIndex = Math.floor(Math.random() * (i + 1));
+            [array[i], array[swapIndex]] = [array[swapIndex], array[i]];
+        }
+    }
+
+    document.querySelectorAll("nav:not([data-adified]), img:not([data-adified])").forEach(target => {
+        target.setAttribute("data-adified", "true");
+        randomize(ads);
+        const wrapper = document.createElement("div");
+        if (target instanceof HTMLImageElement) {
+            target.parentElement?.replaceChild(wrapper, target);
+        } else {
+            target.parentNode?.append(wrapper);
+        }
+        const numRows = (wrapper.offsetHeight / 250);
+        const numCols = (wrapper.offsetWidth / 250);
+        for (let i = 0; i < (numRows > 0 ? numRows : 1); i++) {
+            const row = document.createElement("div");
+            wrapper.className = "vc-ads-wrapper";
+            wrapper.append(row);
+            for (let j = 0; j < (numCols > 0 ? numCols : 1); j++) {
+                const ad = ads[i + j];
+                const link = document.createElement("a");
+                link.href = ad.href ?? "#";
+
+                link.className = "vc-ads-item";
+
+                const img = document.createElement("img");
+                img.src = ad.image;
+                img.setAttribute("data-adified", "true");
+
+                const caption = document.createElement("p");
+                caption.textContent = ad.title;
+
+                link.append(img, caption);
+                row.append(link);
+            }
+        }
+    });
+    setTimeout(addAds, 1000);
+}
+
+export default definePlugin({
+    name: "AdsEverywhere",
+    description: "See ads everywhere, so Discord feels like a real website.",
```

Comment:
```suggestion
    description: "See ads everywhere, so Discord feels like a real website.",
    required: true,
```

think you forgot this

Reply:
Google moment